00-Basics/04-OrderThree/ordre.cpp

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
+using namespace std;
+
+
+
+int main()
+{
+ int a, b, c, temp;
+    
+    cout << "Enter the three numbers : ";
+    cin >> a >> b>> c;
+ 
+while(a>c||a>b||b>c)
+{
+    if(a>b)
+    {
+        temp=a;
+        a=b;
+        b=temp;
```

Comment:
correct solution, so it is ok. One bit of "ugliness" is that the "swap logic" is implemented twice. Once for swapping `a` and `b`, and a little bit below for swapping `b` and `c`. It would be nice, if you could rewrite it in a way, that you put this logic into a function and just call `swap(a,b)` here and `swap(b,c)` below. It is generally bad to implement the same logic more than once redundantly.